backend/app/initial_data.py

```diff
@@ -117,7 +118,10 @@ def wait_for_app():
 
 if __name__ == "__main__":
     print("Creating initial data...")
-    wait_for_app()
+    skip_wait = len(argv) > 1
+
+    if not skip_wait:
+        wait_for_app()
```

Comment:
what's this bit all about?

Reply:
When running from the cli and against a remote host you've tunnelled too - you don't want to wait for the app as it will be the wrong config

Reply:
I meant more that this change looks like "chuck any old garbage on the command line to skip checks"...

Reply:
I also can't immediately see how it has an effect

Reply:
@joel-wright Also writing this up as part of `navigator-scripts` now